batch-service/src/main/java/com/revature/batch/controller/BatchCrudController.java

```diff
@@ -0,0 +1,93 @@
+package com.revature.batch.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.batch.bean.Batch;
+import com.revature.batch.exception.BatchUpdateException;
+import com.revature.batch.exception.NoBatchException;
+import com.revature.batch.service.BatchService;
+
+/**
+ * 
+ * @author Josh Boudreau, Sonam Sherpa, Marko Miocic
+ * Batch: 1802-Feb12-java-matt
+ * 
+ * handles ZUUL endpoint /batches
+ *
+ */
+@RestController
```

Comment:
Missing a (value="") mapping here?

Reply:
Does not need a value if it relates to /batches right?

Reply:
awesome! putting it through now! 🦈